docker/docker-compose.2204.510.yaml

```
@@ -0,0 +1,16 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: vscode-swift:22.04-5.10
+    build:
+      args:
+        ubuntu_version: "focal"
```

```suggestion
        ubuntu_version: "jammy"
```

Ubuntu 22.04 should be `jammy`